Base/res/js/Spreadsheet/runtime.js

```diff
@@ -385,7 +385,11 @@ now.__documentation = JSON.stringify({
     argc: 0,
     argnames: [],
     doc: "Returns a Date instance for the current moment",
-    examples: {},
+    examples: {
+        "now()": "Returns an instance of a Date object",
```

Comment:
Isn't this example a bit redundant? It's exactly the description plus the function

Reply:
It is, I removed it. I wasn't sure if the base function should always be included in examples